modules/thunder_article/modules/thunder_fia/templates/user--facebook-instant-articles-rss.html.twig

```diff
@@ -0,0 +1 @@
+<a>{{ user.displayName }}</a>
```

Comment:
The problem is, that in the default configuration, the User entity doesn't have any name associated. Only a picture and "Member for" would be displayed.